### PR TITLE
0.1.2 release

### PR DESCRIPTION
**Pull request by MaikuB**

### CODE REVIEW

File: device_calendar/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 0.1.2 - 16th May 2019
```
Comment (mkieres) [inline device_calendar/CHANGELOG.md]: missed one '#' or is it intentional?

File: device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/CalendarDelegate.kt [hunk 2]
```diff
@@ -77,19 +77,14 @@ public class CalendarDelegate : PluginRegistry.RequestPermissionsResultListener
...
             return false
         }
 
         val cachedValues: CalendarMethodsParametersCacheModel? = _cachedParametersMap[requestCode]
         if (cachedValues == null) {
-            // Another situation that theoretically should never happen.
-            // No exceptions, this should not happen, ever
-            //
-            // There's nothing that can be done at this stage, besides finishing gracefully
+            // unlikely scenario where another plugin is potentially using the same request code but it's not one we are tracking so return to
```
Comment (mkieres) [inline device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/CalendarDelegate.kt]: The comment doesn't seem to be finished (cuts off at "so return to..." or is this just the github diff tool that's fooling me?

Comment (MaikuB) [reply device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/CalendarDelegate.kt]: it's in the next line but there's a typo (should be indicate not indicator)

File: device_calendar/CHANGELOG.md (hunk 1 above)
Comment (MaikuB) [reply device_calendar/CHANGELOG.md]: nope missed will fix

**Pull request merged**